lte/protos/session_manager.proto

```diff
@@ -910,8 +910,6 @@ message SmContextVoid {
   string output = 1;
 }
 
-
-
```

Comment:
if not changes on session_manager.protos, can you remove the changes on this file?

Reply:
Yes. removed the file.